webapp/src/selectors.ts

```diff
@@ -42,6 +42,13 @@ export const voiceConnectedUsersInChannel = (state: GlobalState, channelID: stri
 
 export const connectedChannelID = (state: GlobalState) => getPluginState(state).connectedChannelID;
 
+const numUsersInConnectedChannel: (state: GlobalState) => number = createSelector(
+    'numUsersInConnectedChannel',
+    connectedChannelID,
+    voiceConnectedChannels,
+    (channelID, channels) => channels ? channels[channelID]?.length || 0 : 0,
```

Comment:
since `voiceConnectedChannels` is initialized as an empty object, I think we can simplify this to:
```suggestion
    (channelID, channels) => channels[channelID]?.length || 0,
```
If this is correct, at some point we could probably clean up the rest of `selectors.ts`.  (or a HW ticket)

Reply:
Yeah, just being extra careful I guess :)